assets/data/recommended_daily_intakes.json

```diff
@@ -0,0 +1,142 @@
+{
```

Comment:
could you suffix the source _eu in the filename ?

Reply:
Fixed in smooth-app branch